packages/dataviews/src/view-table.js

```diff
@@ -200,6 +201,11 @@ function HeaderMenu( { field, view, onChangeView } ) {
 												<DropdownMenuItemLabel>
 													{ element.label }
 												</DropdownMenuItemLabel>
+												{ !! element.description && (
```

Comment:
Thanks for implementing the description also for the table filters, despite this layout not being used in patterns.